pkg/config/config.go

```diff
@@ -1361,8 +1361,7 @@ func ParseConfiguration(_ context.Context, configurationFilePath string, opts ..
 	if cfg.Test != nil {
 		cfg.Test.Environment.Accounts.Groups = append(cfg.Test.Environment.Accounts.Groups, grp)
 	}
-	for i := range cfg.Subpackages {
-		sub := &cfg.Subpackages[i]
+	for _, sub := range cfg.Subpackages {
```

Comment:
Awesome, thanks for this fixup, my attempt at iterating over the range of subpackages was some monkey-see, monkey-do code.